## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 418699`
Commit: `8e72dd58f9d8fa555281659b736d44152841a19b`

---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>bagels</li>
    <li>bagels.dist</li>
    <li>memray</li>
    <li>memray.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 75 packages built:</summary>
  <ul>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>dooit</li>
    <li>dooit-extras</li>
    <li>dooit-extras.dist</li>
    <li>dooit.dist</li>
    <li>frogmouth</li>
    <li>frogmouth.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>isd</li>
    <li>isd.dist</li>
    <li>isponsorblocktv</li>
    <li>isponsorblocktv.dist</li>
    <li>mutmut</li>
    <li>mutmut.dist</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>posting</li>
    <li>posting.dist</li>
    <li>python312Packages.manifestoo</li>
    <li>python312Packages.manifestoo.dist</li>
    <li>python312Packages.pytest-textual-snapshot</li>
    <li>python312Packages.pytest-textual-snapshot.dist</li>
    <li>python312Packages.textual</li>
    <li>python312Packages.textual-autocomplete</li>
    <li>python312Packages.textual-autocomplete.dist</li>
    <li>python312Packages.textual-dev</li>
    <li>python312Packages.textual-dev.dist</li>
    <li>python312Packages.textual-fastdatatable</li>
    <li>python312Packages.textual-fastdatatable.dist</li>
    <li>python312Packages.textual-serve</li>
    <li>python312Packages.textual-serve.dist</li>
    <li>python312Packages.textual-slider</li>
    <li>python312Packages.textual-slider.dist</li>
    <li>python312Packages.textual-textarea</li>
    <li>python312Packages.textual-textarea.dist</li>
    <li>python312Packages.textual-universal-directorytree</li>
    <li>python312Packages.textual-universal-directorytree.dist</li>
    <li>python312Packages.textual.dist</li>
    <li>python313Packages.manifestoo</li>
    <li>python313Packages.manifestoo.dist</li>
    <li>python313Packages.pytest-textual-snapshot</li>
    <li>python313Packages.pytest-textual-snapshot.dist</li>
    <li>python313Packages.textual</li>
    <li>python313Packages.textual-autocomplete</li>
    <li>python313Packages.textual-autocomplete.dist</li>
    <li>python313Packages.textual-dev</li>
    <li>python313Packages.textual-dev.dist</li>
    <li>python313Packages.textual-fastdatatable</li>
    <li>python313Packages.textual-fastdatatable.dist</li>
    <li>python313Packages.textual-serve</li>
    <li>python313Packages.textual-serve.dist</li>
    <li>python313Packages.textual-slider</li>
    <li>python313Packages.textual-slider.dist</li>
    <li>python313Packages.textual-textarea</li>
    <li>python313Packages.textual-textarea.dist</li>
    <li>python313Packages.textual-universal-directorytree</li>
    <li>python313Packages.textual-universal-directorytree.dist</li>
    <li>python313Packages.textual.dist</li>
    <li>rexi</li>
    <li>rexi.dist</li>
    <li>rich-cli</li>
    <li>rich-cli.dist</li>
    <li>smassh</li>
    <li>smassh.dist</li>
    <li>streamcontroller</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>toolong</li>
    <li>toolong.dist</li>
    <li>upiano</li>
    <li>upiano.dist</li>
    <li>wsrepl</li>
    <li>wsrepl.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>bagels</li>
    <li>bagels.dist</li>
    <li>memray</li>
    <li>memray.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 75 packages built:</summary>
  <ul>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>dooit</li>
    <li>dooit-extras</li>
    <li>dooit-extras.dist</li>
    <li>dooit.dist</li>
    <li>frogmouth</li>
    <li>frogmouth.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>isd</li>
    <li>isd.dist</li>
    <li>isponsorblocktv</li>
    <li>isponsorblocktv.dist</li>
    <li>mutmut</li>
    <li>mutmut.dist</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>posting</li>
    <li>posting.dist</li>
    <li>python312Packages.manifestoo</li>
    <li>python312Packages.manifestoo.dist</li>
    <li>python312Packages.pytest-textual-snapshot</li>
    <li>python312Packages.pytest-textual-snapshot.dist</li>
    <li>python312Packages.textual</li>
    <li>python312Packages.textual-autocomplete</li>
    <li>python312Packages.textual-autocomplete.dist</li>
    <li>python312Packages.textual-dev</li>
    <li>python312Packages.textual-dev.dist</li>
    <li>python312Packages.textual-fastdatatable</li>
    <li>python312Packages.textual-fastdatatable.dist</li>
    <li>python312Packages.textual-serve</li>
    <li>python312Packages.textual-serve.dist</li>
    <li>python312Packages.textual-slider</li>
    <li>python312Packages.textual-slider.dist</li>
    <li>python312Packages.textual-textarea</li>
    <li>python312Packages.textual-textarea.dist</li>
    <li>python312Packages.textual-universal-directorytree</li>
    <li>python312Packages.textual-universal-directorytree.dist</li>
    <li>python312Packages.textual.dist</li>
    <li>python313Packages.manifestoo</li>
    <li>python313Packages.manifestoo.dist</li>
    <li>python313Packages.pytest-textual-snapshot</li>
    <li>python313Packages.pytest-textual-snapshot.dist</li>
    <li>python313Packages.textual</li>
    <li>python313Packages.textual-autocomplete</li>
    <li>python313Packages.textual-autocomplete.dist</li>
    <li>python313Packages.textual-dev</li>
    <li>python313Packages.textual-dev.dist</li>
    <li>python313Packages.textual-fastdatatable</li>
    <li>python313Packages.textual-fastdatatable.dist</li>
    <li>python313Packages.textual-serve</li>
    <li>python313Packages.textual-serve.dist</li>
    <li>python313Packages.textual-slider</li>
    <li>python313Packages.textual-slider.dist</li>
    <li>python313Packages.textual-textarea</li>
    <li>python313Packages.textual-textarea.dist</li>
    <li>python313Packages.textual-universal-directorytree</li>
    <li>python313Packages.textual-universal-directorytree.dist</li>
    <li>python313Packages.textual.dist</li>
    <li>rexi</li>
    <li>rexi.dist</li>
    <li>rich-cli</li>
    <li>rich-cli.dist</li>
    <li>smassh</li>
    <li>smassh.dist</li>
    <li>streamcontroller</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>toolong</li>
    <li>toolong.dist</li>
    <li>upiano</li>
    <li>upiano.dist</li>
    <li>wsrepl</li>
    <li>wsrepl.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>bagels</li>
    <li>bagels.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 72 packages built:</summary>
  <ul>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>dooit</li>
    <li>dooit-extras</li>
    <li>dooit-extras.dist</li>
    <li>dooit.dist</li>
    <li>frogmouth</li>
    <li>frogmouth.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>isponsorblocktv</li>
    <li>isponsorblocktv.dist</li>
    <li>mutmut</li>
    <li>mutmut.dist</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>posting</li>
    <li>posting.dist</li>
    <li>python312Packages.manifestoo</li>
    <li>python312Packages.manifestoo.dist</li>
    <li>python312Packages.pytest-textual-snapshot</li>
    <li>python312Packages.pytest-textual-snapshot.dist</li>
    <li>python312Packages.textual</li>
    <li>python312Packages.textual-autocomplete</li>
    <li>python312Packages.textual-autocomplete.dist</li>
    <li>python312Packages.textual-dev</li>
    <li>python312Packages.textual-dev.dist</li>
    <li>python312Packages.textual-fastdatatable</li>
    <li>python312Packages.textual-fastdatatable.dist</li>
    <li>python312Packages.textual-serve</li>
    <li>python312Packages.textual-serve.dist</li>
    <li>python312Packages.textual-slider</li>
    <li>python312Packages.textual-slider.dist</li>
    <li>python312Packages.textual-textarea</li>
    <li>python312Packages.textual-textarea.dist</li>
    <li>python312Packages.textual-universal-directorytree</li>
    <li>python312Packages.textual-universal-directorytree.dist</li>
    <li>python312Packages.textual.dist</li>
    <li>python313Packages.manifestoo</li>
    <li>python313Packages.manifestoo.dist</li>
    <li>python313Packages.pytest-textual-snapshot</li>
    <li>python313Packages.pytest-textual-snapshot.dist</li>
    <li>python313Packages.textual</li>
    <li>python313Packages.textual-autocomplete</li>
    <li>python313Packages.textual-autocomplete.dist</li>
    <li>python313Packages.textual-dev</li>
    <li>python313Packages.textual-dev.dist</li>
    <li>python313Packages.textual-fastdatatable</li>
    <li>python313Packages.textual-fastdatatable.dist</li>
    <li>python313Packages.textual-serve</li>
    <li>python313Packages.textual-serve.dist</li>
    <li>python313Packages.textual-slider</li>
    <li>python313Packages.textual-slider.dist</li>
    <li>python313Packages.textual-textarea</li>
    <li>python313Packages.textual-textarea.dist</li>
    <li>python313Packages.textual-universal-directorytree</li>
    <li>python313Packages.textual-universal-directorytree.dist</li>
    <li>python313Packages.textual.dist</li>
    <li>rexi</li>
    <li>rexi.dist</li>
    <li>rich-cli</li>
    <li>rich-cli.dist</li>
    <li>smassh</li>
    <li>smassh.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>toolong</li>
    <li>toolong.dist</li>
    <li>upiano</li>
    <li>upiano.dist</li>
    <li>wsrepl</li>
    <li>wsrepl.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>bagels</li>
    <li>bagels.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 72 packages built:</summary>
  <ul>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>dooit</li>
    <li>dooit-extras</li>
    <li>dooit-extras.dist</li>
    <li>dooit.dist</li>
    <li>frogmouth</li>
    <li>frogmouth.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>isponsorblocktv</li>
    <li>isponsorblocktv.dist</li>
    <li>mutmut</li>
    <li>mutmut.dist</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>posting</li>
    <li>posting.dist</li>
    <li>python312Packages.manifestoo</li>
    <li>python312Packages.manifestoo.dist</li>
    <li>python312Packages.pytest-textual-snapshot</li>
    <li>python312Packages.pytest-textual-snapshot.dist</li>
    <li>python312Packages.textual</li>
    <li>python312Packages.textual-autocomplete</li>
    <li>python312Packages.textual-autocomplete.dist</li>
    <li>python312Packages.textual-dev</li>
    <li>python312Packages.textual-dev.dist</li>
    <li>python312Packages.textual-fastdatatable</li>
    <li>python312Packages.textual-fastdatatable.dist</li>
    <li>python312Packages.textual-serve</li>
    <li>python312Packages.textual-serve.dist</li>
    <li>python312Packages.textual-slider</li>
    <li>python312Packages.textual-slider.dist</li>
    <li>python312Packages.textual-textarea</li>
    <li>python312Packages.textual-textarea.dist</li>
    <li>python312Packages.textual-universal-directorytree</li>
    <li>python312Packages.textual-universal-directorytree.dist</li>
    <li>python312Packages.textual.dist</li>
    <li>python313Packages.manifestoo</li>
    <li>python313Packages.manifestoo.dist</li>
    <li>python313Packages.pytest-textual-snapshot</li>
    <li>python313Packages.pytest-textual-snapshot.dist</li>
    <li>python313Packages.textual</li>
    <li>python313Packages.textual-autocomplete</li>
    <li>python313Packages.textual-autocomplete.dist</li>
    <li>python313Packages.textual-dev</li>
    <li>python313Packages.textual-dev.dist</li>
    <li>python313Packages.textual-fastdatatable</li>
    <li>python313Packages.textual-fastdatatable.dist</li>
    <li>python313Packages.textual-serve</li>
    <li>python313Packages.textual-serve.dist</li>
    <li>python313Packages.textual-slider</li>
    <li>python313Packages.textual-slider.dist</li>
    <li>python313Packages.textual-textarea</li>
    <li>python313Packages.textual-textarea.dist</li>
    <li>python313Packages.textual-universal-directorytree</li>
    <li>python313Packages.textual-universal-directorytree.dist</li>
    <li>python313Packages.textual.dist</li>
    <li>rexi</li>
    <li>rexi.dist</li>
    <li>rich-cli</li>
    <li>rich-cli.dist</li>
    <li>smassh</li>
    <li>smassh.dist</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>toolong</li>
    <li>toolong.dist</li>
    <li>upiano</li>
    <li>upiano.dist</li>
    <li>wsrepl</li>
    <li>wsrepl.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>bagels</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built bagels-0.3.8-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/bagels-0.3.8...OK
Repacking wheel as ./bagels-0.3.8-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bagels-0.3.8-py3-none-any.whl
  - frozenlist==1.5.0 not satisfied by version 1.6.0</pre>
</details>
<details>
<summary>memray</summary>
<pre>=========================== short test summary info ============================
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_basic - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_basic_node_selected_not_leaf - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_basic_node_selected_leaf - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_two_chains - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_two_chains_after_expanding_second - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_hide_import_system - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_show_uninteresting - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_show_uninteresting_and_hide_import_system - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_select_screen - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_allocations_of_different_sizes - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_biggest_allocations - AssertionError: assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_basic[narrow-terminal-focus-header-long-snapshots] - AssertionError: assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_basic[wide-terminal-long-snapshots] - assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_basic[very-wide-terminal-short-snapshots] - assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_pause[wider-terminal] - assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_gradient - assert False
FAILED tests/unit/test_tui_reporter.py::test_merge_threads - assert False
FAILED tests/unit/test_tui_reporter.py::test_unmerge_threads - assert False
= 18 failed, 504 passed, 8 skipped, 4 deselected, 34 warnings in 63.77s (0:01:03) =</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>bagels</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built bagels-0.3.8-py3-none-any.whl
Finished creating a wheel...
/build/source/dist /build/source
Unpacking to: unpacked/bagels-0.3.8...OK
Repacking wheel as ./bagels-0.3.8-py3-none-any.whl...OK
/build/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bagels-0.3.8-py3-none-any.whl
  - frozenlist==1.5.0 not satisfied by version 1.6.0</pre>
</details>
<details>
<summary>memray</summary>
<pre>=========================== short test summary info ============================
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_basic - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_basic_node_selected_not_leaf - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_basic_node_selected_leaf - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_two_chains - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_two_chains_after_expanding_second - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_hide_import_system - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_show_uninteresting - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_show_uninteresting_and_hide_import_system - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_select_screen - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_allocations_of_different_sizes - AssertionError: assert False
FAILED tests/unit/test_tree_reporter.py::TestTUILooks::test_biggest_allocations - AssertionError: assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_basic[narrow-terminal-focus-header-long-snapshots] - AssertionError: assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_basic[wide-terminal-long-snapshots] - assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_basic[very-wide-terminal-short-snapshots] - assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_pause[wider-terminal] - assert False
FAILED tests/unit/test_tui_reporter.py::test_tui_gradient - assert False
FAILED tests/unit/test_tui_reporter.py::test_merge_threads - assert False
FAILED tests/unit/test_tui_reporter.py::test_unmerge_threads - assert False
= 18 failed, 504 passed, 8 skipped, 4 deselected, 90 warnings in 81.24s (0:01:21) =</pre>
</details>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>bagels</summary>
<pre>Running phase: patchPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built bagels-0.3.8-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-bagels-0.3.8.drv-0/source/dist /private/tmp/nix-build-bagels-0.3.8.drv-0/source
Unpacking to: unpacked/bagels-0.3.8...OK
Repacking wheel as ./bagels-0.3.8-py3-none-any.whl...OK
/private/tmp/nix-build-bagels-0.3.8.drv-0/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bagels-0.3.8-py3-none-any.whl
  - frozenlist==1.5.0 not satisfied by version 1.6.0</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>bagels</summary>
<pre>Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built bagels-0.3.8-py3-none-any.whl
Finished creating a wheel...
/private/tmp/nix-build-bagels-0.3.8.drv-0/source/dist /private/tmp/nix-build-bagels-0.3.8.drv-0/source
Unpacking to: unpacked/bagels-0.3.8...OK
Repacking wheel as ./bagels-0.3.8-py3-none-any.whl...OK
/private/tmp/nix-build-bagels-0.3.8.drv-0/source
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for bagels-0.3.8-py3-none-any.whl
  - frozenlist==1.5.0 not satisfied by version 1.6.0</pre>
</details>
</details>

--- a/pkgs/by-name/me/memray/package.nix
+++ b/pkgs/by-name/me/memray/package.nix
@@ -58,6 +58,14 @@ python3Packages.buildPythonApplication rec {
     # Import issue
     "test_header_allocator"
     "test_hybrid_stack_of_allocations_inside_ceval"
+
+    # The following snapshot tests started failing since updating textual to 3.5.0
+    "TestTUILooks"
+    "test_merge_threads"
+    "test_tui_basic"
+    "test_tui_gradient"
+    "test_tui_pause"
+    "test_unmerge_threads"
   ];
 
   disabledTestPaths = [

--- a/pkgs/development/python-modules/textual/default.nix
+++ b/pkgs/development/python-modules/textual/default.nix
@@ -29,14 +29,14 @@
 
 buildPythonPackage rec {
   pname = "textual";
-  version = "3.4.0";
+  version = "3.5.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "Textualize";
     repo = "textual";
     tag = "v${version}";
-    hash = "sha256-oI0W4KjGHmAGsPFBT2yMWDOLolNPkNOBcLv1K/+dqMc=";
+    hash = "sha256-mr/pzW6EhB+STtVHW1a/+CivzPINHEvvYWnCizycjeo=";
   };
 
   build-system = [ poetry-core ];

